great_expectations/expectations/metrics/map_metric_provider.py

```diff
@@ -1844,9 +1857,8 @@ def _pandas_map_condition_index(
         result_format=result_format,
         execution_engine=execution_engine,
         metrics=metrics,
-        expectation_domain_column_name=expectation_domain_column_name,
+        expectation_domain_column_list=domain_column_name_list,
```

Comment:
This is the biggest change in this PR. Going from a single `column_name` to a `column_name_list`